cpython-unix/extension-modules.yml

```diff
@@ -558,6 +591,18 @@ _testinternalcapi:
     - Include/internal
   sources:
     - _testinternalcapi.c
+  includes-conditional:
+    - path: _testinternalcapi/parts.h
+      minimum-python-version: "3.13"
+  sources-conditional:
+    - source: _testinternalcapi/pytime.c
+      minimum-python-version: "3.13"
+    - source: _testinternalcapi/set.c
+      minimum-python-version: "3.13"
+    - source: _testinternalcapi/test_critical_sections.c
+      minimum-python-version: "3.13"
+    - source: _testinternalcapi/test_lock.c
+      minimum-python-version: "3.13"
```

Comment:
These are "missing" during validation. Is there somewhere we need to ensure they're populated in the build?

Reply:
Got past this with https://github.com/zanieb/python-build-standalone/pull/5/commits/e153e1087724491c94b465df2d005c0f141d0959